### PR TITLE
WEBUI-879: select all remains active when unselecting individual docu…

### PR DESCRIPTION
**Pull request by rakeshkumar1019**
https://jira.nuxeo.com/browse/WEBUI-879

### CODE REVIEW

**Review by swarnadipa-dev — DISMISSED** [review]
looks good

**Review by SnehalNaik-DEV — DISMISSED** [review]
Looks Good.

**Review by swarnadipa-dev — DISMISSED** [review]
Looks good

**Review by GuruprasadAnk1996 — DISMISSED** [review]

File: elements/nuxeo-results/nuxeo-results.js [hunk 1]
```diff
@@ -482,9 +483,10 @@ Polymer({
...
       return;
     }
-
-    if (this.view.selectAllActive) {
+    if (this._excludedDocs === 0 && this.view.selectAllActive) {
       this.clearSelection();
+    } else if (this._excludedDocs > 0 && this.view.selectAllActive) {
+      this.selectAll();
     } else {
       this.selectAll();
```
Comment (rajatgoel-gl) [inline elements/nuxeo-results/nuxeo-results.js:489]: Kindly recheck conditions. It looks like unnecessary conditions have been intoduced.

File: elements/nuxeo-results/nuxeo-results.js [hunk 2]
```diff
@@ -745,6 +747,16 @@ Polymer({
     }
   },
 
+  _isChecked(selectAllActive, _excludedDocs) {
+    if (selectAllActive && _excludedDocs <= 0) {
+      return true;
+    }
+    if (selectAllActive && _excludedDocs > 0) {
+      return false;
+    }
```
Comment (rajatgoel-gl) [inline elements/nuxeo-results/nuxeo-results.js]: Kindly recheck conditions over here. Line 754-756 is not required.

**Review by rajatgoel-gl — CHANGES_REQUESTED** [review]

File: elements/nuxeo-results/nuxeo-results.js (hunk 1 above)
Comment (rakeshkumar1019) [reply elements/nuxeo-results/nuxeo-results.js:489]: Thank you @rajatgoel-gl did the changes.

**Review by swarnadipa-dev — APPROVED** [review]
looks good

**Review by GuruprasadAnk1996 — APPROVED** [review]

**Pull request closed without merging**